bootstrap/051_materialization_status.sql

```diff
@@ -0,0 +1,159 @@
+            ('QUERY_HISTORY', 'REPORTING', 'ENRICHED_QUERY_HISTORY', 'QUERY_HISTORY_MAINTENANCE'),
+            ('WAREHOUSE_EVENTS_HISTORY', 'REPORTING', 'WAREHOUSE_SESSIONS', 'WAREHOUSE_EVENTS_MAINTENANCE'),
+            ('WAREHOUSE_EVENTS_HISTORY', 'REPORTING', 'CLUSTER_SESSIONS', 'WAREHOUSE_EVENTS_MAINTENANCE'),
+            ('SERVERLESS_TASK_HISTORY', 'REPORTING', 'SERVERLESS_TASK_HISTORY', 'SIMPLE_DATA_EVENTS_MAINTENANCE'),
+            ('TASK_HISTORY', 'REPORTING', 'TASK_HISTORY', 'SIMPLE_DATA_EVENTS_MAINTENANCE'),
+            ('SESSIONS', 'REPORTING', 'SESSIONS', 'SIMPLE_DATA_EVENTS_MAINTENANCE'),
+            ('WAREHOUSE_METERING_HISTORY', 'REPORTING', 'WAREHOUSE_METERING_HISTORY', 'SIMPLE_DATA_EVENTS_MAINTENANCE'),
+            ('LOGIN_HISTORY', 'REPORTING', 'LOGIN_HISTORY', 'SIMPLE_DATA_EVENTS_MAINTENANCE'),
+            ('HYBRID_TABLE_USAGE_HISTORY', 'REPORTING', 'HYBRID_TABLE_USAGE_HISTORY', 'SIMPLE_DATA_EVENTS_MAINTENANCE'),
+            ('MATERIALIZED_VIEW_REFRESH_HISTORY', 'REPORTING', 'MATERIALIZED_VIEW_REFRESH_HISTORY', 'SIMPLE_DATA_EVENTS_MAINTENANCE')
+        ) as t(table_name, user_schema, user_view, task_name)
+    ), task_history as (
+        -- The history rows generated every time a task is run and the table that task is managing
+        select $1 as run, $2 as success, $3 as input, $4 as output, $5 as table_name, $6 as task_name, IFF(INPUT is null, 'FULL', 'INCREMENTAL') as kind FROM (
+            select run::TIMESTAMP_LTZ, success, input, output, 'QUERY_HISTORY', 'QUERY_HISTORY_MAINTENANCE'   from internal.task_query_history
+            union all
+            select run::TIMESTAMP_LTZ, success, input, output, 'WAREHOUSE_EVENTS_HISTORY', 'WAREHOUSE_EVENTS_MAINTENANCE' from internal.task_warehouse_events
+            union all
+            select run::TIMESTAMP_LTZ, success, input, output, table_name, 'SIMPLE_DATA_EVENTS_MAINTENANCE' from internal.task_simple_data_events
+        )
+    ), sf_task_history as (
+        -- Our TASK_HISTORY view is updated with a 3hour delay. Limit how far we back we look in the UDTF.
+        select name as task_name, query_id, graph_run_group_id, state, scheduled_time, QUERY_START_TIME,
+        from table(information_schema.task_history(SCHEDULED_TIME_RANGE_START => TIMESTAMPADD(HOUR, -3, current_timestamp()), result_limit => 10000))
+        WHERE database_name in (select current_database()) and schema_name = 'TASKS'
+        union
+        select name as task_name, query_id, graph_run_group_id, state, scheduled_time, QUERY_START_TIME,
+        from reporting.task_history
+        WHERE database_name in (select current_database()) and schema_name = 'TASKS'
+    ), completed_tasks as (
+        select * from sf_task_history where internal.task_complete(state)
+    ), pending_tasks as (
+        select * from sf_task_history where internal.task_pending(state)
+    ), fullmat as (
+        SELECT th.table_name, th.task_name, run, output['end']::TIMESTAMP_LTZ as end, success,
```

Comment:
Thinking more, the intent was likely that we're reporting the range of data materialized, not the duration of the materialization task (user doesn't care about that)